lizard_connector/connector.py

```diff
@@ -36,9 +36,20 @@
     import urllib.request as urllib_request
     from urllib.request import urlopen
 
-
+DEFAULT_API_VERSION = '3'
 ASYNC_POLL_TIME = 1
 ASYNC_POLL_TIME_INCREASE = 1.5
+ADDITIONAL_ENDPOINTS_V3 = (
+    'raster_aggregates',
+)
+DETAIL_ENDPOINTS = (
+    'rasters',
+    'timeseries',
+    'opticalfibers'
+)
```

Comment:
Zijn dit de enige die werken?

Reply:
De naam is verkeerd, het gaat om endpoints met de vorm: `{base_url}/{endpoint}/{uuid}/data`. Dat zijn deze 3 en die hebben allemaal een uuid als `pk`.